library/core/src/marker.rs

```diff
@@ -923,3 +923,14 @@ mod copy_impls {
     #[stable(feature = "rust1", since = "1.0.0")]
     impl<T: ?Sized> Copy for &T {}
 }
+
+/// A common trait implemented by all function pointers.
+#[unstable(feature = "fn_ptr_trait", issue = "none", reason = "recently added")]
```

Comment:
`issue = "none"` normally implies that this is not a feature on its way to stabilization, in which case it should have something like `reason = "internal …"`.

If this is something we might want to stabilize in the future, it should have a tracking issue, and probably just no `reason =` attribute.